x-pack/legacy/plugins/ml/public/jobs/new_job_new/pages/components/pick_fields_step/components/summary_count_field/summary_count_field.tsx

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC, useContext, useEffect, useState } from 'react';
+
+import { SummaryCountFieldSelect } from './summary_count_field_select';
+import { JobCreatorContext } from '../../../job_creator_context';
+import { newJobCapsService } from '../../../../../../../services/new_job_capabilities_service';
+import {
+  MultiMetricJobCreator,
+  isMultiMetricJobCreator,
+  PopulationJobCreator,
+  isPopulationJobCreator,
+  AdvancedJobCreator,
+  isAdvancedJobCreator,
+} from '../../../../../common/job_creator';
+import { Description } from './description';
+
+export const SummaryCountField: FC = () => {
+  const { jobCreator: jc, jobCreatorUpdate, jobCreatorUpdated } = useContext(JobCreatorContext);
+  if (
```

Comment:
Again, as mentioned in the other component, returning `null` here before calling all hooks isn't valid. All hooks need to be called on every render.